docker/dbt/Dockerfile

```diff
@@ -11,7 +11,7 @@ RUN apt-get update && \
     apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/
 RUN pip3 install --upgrade pip
 RUN mkdir ${DBT_DIR}
-RUN pip install dbt-trino==1.3
+RUN pip install 'dbt-trino~=1.3.0'
```

Comment:
'dbt-trino~=1.3.*'

Reply:
I tried the command, however it didn't work. Above command seems to have the same result.

```
❯ pip install 'dbt-trino~=1.3.*'
ERROR: Invalid requirement: 'dbt-trino~=1.3.*'
```

Reply:
If you use the star, then we need to use `==`, so the command would be `pip install 'dbt-trino==1.3.*'`. Anyway it gives the same result.